bokeh/models/tiles.py

```diff
@@ -26,11 +26,11 @@ class TileSource(PlotObject):
     the maximum zoom level for the tile layer. This is the most "zoomed-in" level.
     """)
 
-    x_origin_offset = Float(default=20037508.34, help="""
+    x_origin_offset = Float(default=None, help="""
```

Comment:
You are setting x_origin_offset to the 20037508.34 number in coffeescript ... why None in Python? similarly for some of the other properties.